plotting/ft_plot_mesh.m

```diff
@@ -139,11 +139,14 @@
 end
 
 % new colors management
-if strcmpi(vertexcolor,'skin') || strcmpi(vertexcolor,'brain') || strcmpi(vertexcolor,'cortex')
-  vertexcolor = eval(vertexcolor);
+if ischar(vertexcolor) && numel(vertexcolor) <= 63 % 63 is the max evaluable string length
+	try vertexcolor = eval(vertexcolor); catch, end
```

Comment:
try-catch is expensive, i.e. it takes quite some time. Could you change it into the following?

if ischar(vertexcolor) && exist([vertexcolor '.m'], 'file')
  vertexcolor = eval(vertexcolor);
end